### PR TITLE
Set up CI.

### PR DESCRIPTION
**Pull request by rasolca**
Set up Jenkins to tests Pull Requests.

### CODE REVIEW

**Comment by rasolca** [comment]
Closes #14.

File: ci/daint-mc_compile.sbatch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -l
+#SBATCH --job-name=DLA-Future-compile-mc
+#SBATCH --nodes=1
+#SBATCH --constraint=mc
+#SBATCH --partition=cscsci
+#SBATCH --time=00:15:00
+
+set -e
+
+printenv
```
Comment (albestro) [inline ci/daint-mc_compile.sbatch]: is it wanted? before changing it with the next command?

File: ci/daint-mc_compile.sbatch [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+mkdir -p $DIR
+cd $DIR
+cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
+      -DDLAF_USE_MKL=1                 \
+      -DHPX_DIR=$EBROOTHPX             \
+      -DBLASPP_ROOT=$EBROOTBLASPP      \
+      -DLAPACKPP_ROOT=$EBROOTLAPACKPP  \
+      -DMPIEXEC_EXECUTABLE="srun"      \
+      -DMPIEXEC_NUMPROC_FLAG="-n "     \
```
Comment (albestro) [inline ci/daint-mc_compile.sbatch]: I think that the space after "-n" is superfluous

File: ci/make-doc.sbatch [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#SBATCH --error=make-doc.out.txt
+
+set -ex
+
+source ${ENV}
+module list
+DIR=build_doc
+
+mkdir -p $DIR
+(
```
Comment (albestro) [inline ci/make-doc.sbatch]: This is not following the same template of the others scripts

**Pull request merged**